auth/src/main/res/values-v26/styles.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:ignore="ResourceName">
+
+    <style name="FirebaseUI.TextInputEditText.EmailField">
+        <item name="android:autofillHints">emailAddress</item>
```

Comment:
Oh, thanks for reminding me! I was going to disable password autofill if smart lock is enabled (because getting a save password prompt twice is really annoying).

As for the email, we could also disable autofill anytime smart lock is enabled because the only way the user could have reached the email field is if they didn't take the hint with smart lock enabled.